src/loginAzurePowerShell.ts

```diff
@@ -0,0 +1,76 @@
+import * as os from 'os';
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+import * as io from '@actions/io';
+
+var psPath: string;
+
+export const initializeAz = async (servicePrincipalId: string, servicePrincipalKey: string, tenantId: string, subscriptionId: string) => {
```

Comment:
Do not use Anonymous function. use named functions. Export statements should be confined towards the end of the file.